docs/website/package.json

```diff
@@ -35,17 +35,13 @@
   "devDependencies": {
     "@11ty/eleventy": "^2.0.1",
     "@11ty/eleventy-dev-server": "^2.0.4",
+    "@exadel/eslint-config-esl": "^5.0.0-beta.36",
```

Comment:
It is not necessary for the current update but I'd suggest adding 
"@exadel/eslint-plugin-esl" as well